### PR TITLE
Add options for constructor, 'getSessionUrl' method and increase ping interval; bump version (closes #16)

### LINKED ISSUE #16: Provide `getSessionUrl` API method, prolong ping interval, extend browser's init options.
These requirments were requested by @AndreyBelym for development browser provider plugin.

UPD. Add ability to disable browser start message.

### PR DESCRIPTION
**Pull request by AndreyBelym**
This PR supersedes #17.
\cc @AlexanderMoskovkin @georgiy-abbasov

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -13,28 +13,49 @@ const PRERUN_SCRIPT_DIR_PATH                        = toAbsPath('./prerun/');
 const DISABLE_COMPATIBILITY_MODE_IE_SCRIPT_FILENAME = 'disable-intranet-compatibility-mode-in-ie.bat';
 
 const WEB_DRIVER_IDLE_TIMEOUT              = 1000;
-const WEB_DRIVER_PING_INTERVAL             = 900;
+const WEB_DRIVER_PING_INTERVAL             = 5 * 60 * 1000;
 const WEB_DRIVER_CONFIGURATION_RETRY_DELAY = 30 * 1000;
 const WEB_DRIVER_CONFIGURATION_RETRIES     = 3;
 const WEB_DRIVER_CONFIGURATION_TIMEOUT     = 9 * 60 * 1000;
+const DEFAULT_DIRECT_DOMAINS               = ['*.google.com', '*.gstatic.com', '*.googleapis.com'];
```
Comment (AlexanderMoskovkin) [inline src/index.js:24]: please add the comment why use these domains by default

**Comment by AndreyBelym** [comment]
FPR, comment has been checked by techwriter.

**Comment by georgiy-abbasov** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**